pepper-apis/src/main/java/org/broadinstitute/ddp/customexport/export/CustomExporter.java

```diff
@@ -47,20 +57,66 @@
 
     // A cache for user auth0 emails, storing (auth0UserId -> email).
     private static final Map<String, String> emailStore = new HashMap<>();
+    private static final String familyIdHeader = "familyId";
```

Comment:
I think all the column names in CSV export use snake_case, so you might want to be consistent and name it `family_id`.

Reply:
I agree with this point and noticed it myself but only after I told the Seqr people that it would be written this way so I think I'm stuck with it...